docs/docs/recipes/protect-your-api/README.md

```diff
@@ -7,26 +7,26 @@ import TabItem from '@theme/TabItem';
 
 # ⚔️ Protect your API
 
-Along with a fluent user sign-in experience, by default, Logto comes with the [API resources](../../references/resources/README.md) authorization service. It will help you protect the private APIs resources from anonymous identities. Let's walk through the following steps and implement the API resource authorization guard of your own using Logto.
+Along with a fluent user sign-in experience, by default, Logto comes with the [API Resources](../../references/resources/README.md) authorization service. It will help you protect the private APIs resources from anonymous identities. Let's walk through the following steps and implement the API Resource authorization guard of your own using Logto.
```

Comment:
the commit message and pr title looks verbose. in docs we just have two scopes now: `(en-US)` and `(zh-CN)`

Reply:
updated and rebase the commit message